tests/Element/TagInputTest.php

```
@@ -127,6 +127,54 @@ public function testRenderWithAriaAttributes(): void
         );
     }
 
+    public function testRenderWithAriaDescribedByTrue(): void
+    {
+        self::assertEquals(
+            <<<HTML
+            <input id="test-id" aria-describedby="test-id-help">
+            HTML,
+            LineEndingNormalizer::normalize(
+                TagInput::tag()->id('test-id')->attributes(['aria-describedby' => true])->render(),
+            ),
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\PHPForge\Support\LineEndingNormalizer' in method 'testRenderWithAriaDescribedByTrue'. (undefined)

(StaticAccess)